pkg/secrets/sealed_secrets.go

```diff
@@ -44,44 +46,72 @@ func replaceSecretsWithSealedSecrets(archivePath string, config map[string][]byt
 			return errors.Wrap(err, "failed to read file")
 		}
 
-		decoded, _, err := decode(contents, nil, nil)
-		if err != nil {
-			return nil
-		}
+		multiDocYaml := bytes.Split(contents, []byte("---\n"))
+		var secrets []byte
+		var nonSecrets []byte
```

Comment:
i'd recommend changing this to this, and then run `bytes.Join(nonSecrets, []byte("\n---\n"))` at the end. same for the secrets slice
```suggestion
		var nonSecrets [][]byte
```

Reply:
resolved with latest push